.gitignore

```diff
@@ -15,3 +15,8 @@ __pycache__/
 result*
 .env-local
 .history
+
+
+#IDE
```

Comment:
Is it really needed in prod commit?

Reply:
@pashkinelfe they can be taken out.

Reply:
It's pretty conventional to have IDE gitignores in any given git repo.

Reply:
Unless there is a strong objection I will leave this in place